tests/DevApps/aspnet-mvc/OwinWebApp/App_Start/Startup.Auth.cs

```diff
@@ -21,7 +19,11 @@ public void ConfigureAuth(IAppBuilder app)
 
             OwinTokenAcquirerFactory factory = TokenAcquirerFactory.GetDefaultInstance<OwinTokenAcquirerFactory>();
 
-            app.AddMicrosoftIdentityWebApp(factory);
+            app.AddMicrosoftIdentityWebApp(factory, updateOptions: (options) =>
+            {
+                options.RedirectUri = "https://localhost:44386/";
```

Comment:
>RedirectUri [](http://example.com/codeflow?start=24&length=11)

This is the option which affects the redirect_uri parameter.

Reply:
I don't think this is needed, it looked to be an app registration issue. Added `https://localhost:44386/` as a redirectURI for this app in the lab. Not sure how it is even working, as this redirectURI was not registered to the app.

Reply:
I found this issue with a dSTS application, this is unrelated with the configuration of the application. This shows how to affect a specific parameter called redirect_uri.